openhtf/io/proto/assembly.proto

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Assembly information is actually explicitly not in-scope for OpenHTF currently, so I'd rather leave this file out for now.